Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -15,9 +15,47 @@
 import ArcGIS
 import SwiftUI
 
-/// Forms allow users to edit information about GIS features.
+/// The `FeatureFormView` component enables users to edit field values of features
+/// in a layer using forms that have been configured externally, either in
+/// the Web Map Viewer or the Fields Maps Designer.
 ///
-/// - Since: 200.4
```

Comment:
Should this have been moved or deleted all together?

Reply:
I was just going to delete it, but it is helpful to have, so I added it back:
![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/5f76ad37-1bff-44d4-bf09-57b36df2ff3f)